source/cloud-native-platform/guides/postgresql-singleserver-to-flexibleserver-migration-portal.html.md.erb

```diff
@@ -0,0 +1,30 @@
+---
+title: Postgresql single server to flexible server migration using Azure Portal
+last_reviewed_on: 2023-04-24
+review_in: 6 months
+weight: 10
+hide_in_navigation: true
+---
+
+# <%= current_page.data.title %>
+
+The Azure Portal can be used to perform an offline migration of a PostgreSQL Single Server instance to PostgreSQL Flexible Server.
+
+This simple method has lower chances of failure and no restrictions in terms of database objects it can handle.
+
+### Limitations
+
+- Will result in downtime for applications.
+
+### Prerequisite 
+
+- Create target Postgresql flexible server using [terraform module](https://github.com/hmcts/terraform-module-postgresql-flexible). Make sure database names and sizes are identical and equal or larger in size respectively.
+- Allowlist all required extensions as shown in [Migrate from Azure Database for PostgreSQL Single Server to Flexible Server](https://learn.microsoft.com/en-us/azure/postgresql/migrate/concepts-single-to-flexible#allow-list-required-extensions). It is important to allowlist the extensions before you initiate a migration using this tool.
+
+### Migration Steps
+
+AzureDevops pipeline as been created to facilitate migration process. Follow instructions in README.MD file located in [repo](https://github.com/hmcts/flexible-server-migration).
+
+Azure [documentation](https://learn.microsoft.com/en-us/azure/postgresql/migrate/how-to-migrate-single-to-flexible-portal#configure-the-migration-task).
+
+Developers need to reach out to platform operation team for database password.
```

Comment:
```suggestion
Developers need to reach out to platform operation team for database password.
```
This isn't needed any more, the devs just need to know the key vault and the name of the secret